### ISSUE TITLE
Error in c++ intellisense for imported constructor

### ISSUE THREAD

**Issue by EdorianDark** [microsoft/vscode-cpptools#5188]
**Type: LanguageService**

**Describe the bug**
- OS and Version: local Windows, remote Linux
- VS Code Version: 1.43.2
- C/C++ Extension Version: 0.26.3
- Other extensions you installed (and if the issue persists after disabling them):  Remote Development, C++ Themes, both from Microsoft
- Does this issue involve using SSH remote to run the extension on a remote machine?: yes
- A clear and concise description of what the bug is, including information about the workspace (i.e. is the workspace a single project or multiple projects, size of the project, etc).

**Steps to reproduce**
Copy following code into a C++ Project 
````c++
class AA {
    public:
    AA(int i = 0){i_ = i;};
    AA(const AA& other): i_(other.i_){}
    private:
    int i_;
};
class BB : public AA{
    public:
    using  AA::AA;
    BB(const BB& other) : AA(other) {}
};
void f(){
    BB b;
}
````

**Expected behavior**
Since the code compiles and looks correct, there should not problems be reported.

**Actual behavior**
In the problems tab, the following message is listed:
``
no default constructor exists for class "BB"
``

**Screenshots**

![image](https://user-images.githubusercontent.com/5777938/77534938-efc5f100-6e99-11ea-8af7-cbe1d5d8c500.png)

**Comment by Colengms** [comment]
Hi @EdorianDark .  Thanks for reporting this.  I'm able to repro with Visual Studio (VS and the C/C++ Extension for VS Code share common code for IntelliSense).  I opened a bug against VS here: https://developercommunity.visualstudio.com/content/problem/963209/c-intellisense-reports-no-default-constructor-desp.html

**Comment by Colengms** [comment]
It looks like we picked up a fix for this issue in 1.0.0.  Closing as fixed.
